### DISCUSSION TITLE
Support for FlipperZero?

### DISCUSSION THREAD

**Discussion by benbuschmann** [ZerBea/hcxdumptool#435] [General]
@ZerBea thoughts on possibility of getting hcxdumptool working for flipper zero dev wifi board (ESP32-S2 MCU)? 

https://docs.flipper.net/development/hardware/wifi-developer-board

**Comment by ZerBea** [comment]
No thoughts due to this reasons:

[FreeTOS](https://www.freertos.org) instead of [Linux](https://www.kernel.org/)
less powerful [hardware](https://en.wikipedia.org/wiki/Flipper_Zero) than a [Raspberry Pi](https://en.wikipedia.org/wiki/Raspberry_Pi) Zero
the [ESP32](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/) requires additional effort

The combination of this reasons makes it necessary to develop everything from scratch. The effort is too high (for me).
